### DISCUSSION TITLE
I don't want to copilot any more.

### DISCUSSION THREAD

**Discussion by HA-SEUNG-JEONG** [community/community#31705] [Copilot Conversations] [answered]
I just received a copy of the payment receipt, can I get a refund within today?

**Comment by davecheney** [comment, marked as answer]
Please contact support, https://support.github.com/contact

**Comment by ZhaoKunLong** [comment]
@HA-SEUNG-JEONG   To be honest, is it useful? How is your experience? Is it suitable for front-end development?
